.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: objective-c  #Travis only recognizes Obj-C, Xcode will handle the fact that this is Swift
+
+# xcode and sdk versions
+osx_image: xcode7
+xcode_sdk: iphonesimulator9.0
```

Comment:
Eventually we can set up a matrix that will run against a lot of different sdks versions.